crypto/vrf/vrf_r2ishiguro.go

```diff
@@ -1,4 +1,5 @@
-// +build !libsodium
+// default: r2ishiguro
```

Comment:
This might be basic for golang, but... what does this `// default:` statement mean in golang?
Does this affect conditional compilation?

Reply:
This is just a comment in `golang`. It is meaningless to build.